config/environments/development.rb

```diff
@@ -37,5 +37,5 @@
 
   # config/environments/production.rb
 
-  Paperclip.options[:command_path] = "/usr/local/bin/"
+  Paperclip.options[:command_path] = "C:/Program Files/ImageMagick-6.9.1-Q16"
```

Comment:
On 6/4/2015 5:20 PM, sweir27 wrote:
>
> In config/environments/development.rb 
> <https://github.com/sweir27/coop_gallery_213/pull/16#discussion_r31766879>:
>
> > @@ -37,5 +37,5 @@
> >
> >    # config/environments/production.rb
> >
> > -  Paperclip.options[:command_path] = "/usr/local/bin/"
> > +  Paperclip.options[:command_path] = "C:/Program Files/ImageMagick-6.9.1-Q16"
>
> When this is done it will no longer show up here. You should fix the 
> path in your |.gitignore| and make sure it is not committed.
>
> To only add certain files to be staged for commit, you can do |git add 
> <filename>| and check which ones are staged by doing |git status|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sweir27/coop_gallery_213/pull/16/files#r31766879>.
>
I reverted this file.